react/components/ProductKitDetails.js

```diff
@@ -55,6 +55,12 @@ export default class ProductKitDetails extends Component {
       price: path(['sku', 'seller', 'commertialOffer', 'Price'], item),
       variant: item.sku.name,
       brand: item.brand,
+
+      /* Optimistic props */
```

Comment:
Why the descriptor? I don't find confusing mixing with other props

Reply:
It's kinda weird to pass, for example, the product image to a buy button...